src/tests/multihost/basic/test_sudo.py

```diff
@@ -62,8 +62,4 @@ def test_refresh_expired_rule(self, multihost,
             (_, _, exit_status) = ssh.execute_cmd('sudo -l')
             assert exit_status == 0
             time.sleep(30)
-            if exit_status != 0:
```

Comment:
I would try to figure out what was the intention.
Might be a bug.

Reply:
I think this is a leftover from a debug session. Those lines are used to print information from the journal in case the command fails. I think we are safe to remove those.